package.json

```diff
@@ -8,6 +8,7 @@
     "build-detox": "detox build",
     "build-user-scripts": "webpack --config webpack.config.js",
     "test-detox": "detox test --take-screenshots all",
+    "lint": "tsc --noEmit && eslint --ext .js,.jsx,.ts,.tsx ./ReactNative/js",
```

Comment:
What about a command to run `prettier`? I use the following commands in multiple projects:

```json
    "format:check": "prettier '{src,test}/**/*.ts' --check",
    "format:fix": "prettier '{src,test}/**/*.ts' --write",
    "lint:check": "tslint --config ./tslint.json --project ./tsconfig.json",
    "lint:fix": "tslint --config ./tslint.json --project ./tsconfig.json --fix",
    "typecheck": "tsc --project ./tsconfig.json",
    "typecheck:watch": "tsc --project ./tsconfig.json --watch",
    "yolo": "npm run format:fix && npm run lint:fix && npm run typecheck",
```

Reply:
yolo 😂

Let's address this in separate PR.

Reply:
Life is too short to run multiple commands :P